Resources/Prototypes/_RMC14/Entities/Structures/Doors/Shutters/shutters.yml

```diff
@@ -0,0 +1,96 @@
+- type: entity
+  parent: CMBaseStructureCorrodible
+  id: RMCShutterBase
+  abstract: true
+  name: shutters
+  description: That looks like it doesn't open easily.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    sprite: _RMC14/Structures/Doors/Shutters/Almayer/shutter.rsi
+    drawdepth: BlastDoors
+    layers:
+    - state: closed
+      map: ["enum.DoorVisualLayers.Base"]
+  - type: AnimationPlayer
+  - type: Physics
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.49,-0.49,0.49,0.49" # don't want this colliding with walls or they won't close
+        density: 100
+        mask:
+        - FullTileMask
+        layer:
+        - AirlockLayer
+  - type: ContainerFill
+    containers:
+      board: [ DoorElectronics ]
+  - type: ContainerContainer
+    containers:
+      board: !type:Container
+  - type: Door
+    openDrawDepth: BlastDoors
+    closedDrawDepth: BlastDoors
+    bumpOpen: false
+    clickOpen: false
+    closeTimeOne: 0.4
+    closeTimeTwo: 0.4
+    openTimeOne: 0.4
+    openTimeTwo: 0.4
+    openingAnimationTime: 1.0
+    closingAnimationTime: 1.0
+    crushDamage:
+      types:
+        Blunt: 5
+    openSound:
+      path: /Audio/_RMC14/Machines/blastdoor.ogg
+    closeSound:
+      path: /Audio/_RMC14/Machines/blastdoor.ogg
+  - type: Appearance
+  - type: UserInterface
+    interfaces:
+      enum.WiresUiKey.Key:
+        type: WiresBoundUserInterface
+  - type: IconSmooth
+    key: walls
+    mode: NoSprite
+  - type: DoorSignalControl
+  - type: DeviceNetwork
+    deviceNetId: Wireless
+    receiveFrequencyId: BasicDevice
+  - type: WirelessNetworkConnection
+    range: 200
+  - type: DeviceLinkSink
+    ports:
+      - Open
+      - Close
+      - Toggle
+  - type: DeviceLinkSource
+    ports:
+    - DoorStatus
+    lastSignals:
+      DoorStatus: false
```

Comment:
```suggestion
```
These probably shouldnt be able to opened with a multitool and a signal button

Reply:
yes and no, they should, but buttons in cmss13 works a little bit differently. They and shutter have the same id.

- shutters:
![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/7cd773c9-6d34-4bb2-adea-66f2876d092c)
- button
![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/2ae6c7d8-4d20-4fec-9dea-54e464052fef)

![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/1d604c8b-a3e9-4c8c-badc-baa5a8dfedb6)

Reply:
Probably need a different system at least for the poddoors then, otherwise players would be able to link some button on the map to the poddoors and bypass them entirely.

Reply:
well, <b>all</b> shutters and poddoors on Solaris have linked buttons.

![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/15b2727a-d78f-4293-bb12-75374ae114c7)
![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/481ac7f5-04d1-4b94-aca6-80e641f525fc)
![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/caf1e73f-87d6-496d-95a2-d2ef60b7db6b)
![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/ef6de74f-d914-4d93-a7e4-30aa6fd99238)
![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/6f891941-0367-4ffc-8701-8f5433ca8901)
![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/a9cdf9ba-ed5c-4ff0-b7cf-6ff52789b1b7)

Reply:
all, but this. this poddoors have standard id (1) and i cant find button linked to them.
![зображення](https://github.com/RMC-14/RMC-14/assets/71643624/7e69d5d4-9405-49a4-a238-a644198a0113)

Reply:
i'l just leave `# TODO` in proto then.